yarn.lock

```diff
@@ -111,6 +111,11 @@
   resolved "https://registry.yarnpkg.com/@types/jasmine/-/jasmine-3.5.10.tgz#a1a41012012b5da9d4b205ba9eba58f6cce2ab7b"
   integrity sha512-3F8qpwBAiVc5+HPJeXJpbrl+XjawGmciN5LgiO7Gv1pl1RHtjoMNqZpqEksaPJW05ViKe8snYInRs6xB25Xdew==
 
+"@types/js-yaml@^4.0.5":
+  version "4.0.5"
+  resolved "https://artifactory.prod.tableautools.com/artifactory/api/npm/tab-npm/@types/js-yaml/-/js-yaml-4.0.5.tgz#738dd390a6ecc5442f35e7f03fa1431353f7e138"
```

Comment:
ah so looks like you have a rc file up the chain or have a config set to use Artifactory. We don't want to use our npm proxy and instead want all of these to be registry.yarnpkg.com since this is an external project.

Reply:
Fixed in latest.  I also added a .yarnrc pointing to registry.yarnpkg.com.